deploy-sandpit.sh

```diff
@@ -1,23 +1,54 @@
+# !/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

No space in a shebang